templates/zerver/help/star-a-message.md

```diff
@@ -3,12 +3,33 @@
 Starring messages is a good way to keep track of important messages, such as
 tasks you need to go back to or documents you reference often.
 
+### Star a message
+
+{start_tabs}
+
 {!message-actions.md!}
 
 1. Click the star (<i class="fa fa-star-o"></i>) icon.
 
-You can access your starred messages by clicking **Starred Messages** in the
-left sidebar, or by [searching](/help/search-for-messages) for `is:starred`.
+{end_tabs}
 
 Starred messages have a filled in star (<i class="fa fa-star"></i>) to
 their right. Click on it to unstar the message.
+
+## Access your starred messages
+
+You can access your starred messages by clicking **Starred Messages** in the
+left sidebar, or by [searching](/help/search-for-messages) for `is:starred`.
```

Comment:
We should probably talk about the use case here; e.g. that this is helpful if your workflow is to star messages on mobile to come back to on the web.

Reply:
There's an unmerged article "reading strategies", where I've added this.

Reply:
fwiw, the top of this article says:
 Starring messages is a good way to keep track of important messages, such as
 tasks you need to go back to or documents you reference often.